test/unit/storage_test.rb

```diff
@@ -149,12 +185,13 @@ def assert_sentinel_connector(client)
     assert_instance_of Redis::Client::Connector::Sentinel, connector
   end
 
-  def assert_client_config(client, host: nil, port: nil, url: nil, sentinels: nil)
+  def assert_client_config(client, host: nil, port: nil, url: nil, sentinels: nil, role: nil)
```

Comment:
I think we can improve the function name: `assert_client_config_is_correct` or similar. Right now it reads like an incomplete sentence.

Also, have you considered passing `attrs_to_check = {}` ? All the params have `nil` as the default value, so it would be equivalent and cleaner, in my opinion.

Reply:
@eguzki yes, from 5 to 6 you go over rubocop default: http://rubocop.readthedocs.io/en/latest/cops_metrics/#metricsparameterlists

Reply:
we can have host and port separated from the rest (because they are used differently) and iterate the rest as a hash - wdyt @eguzki?

Reply:
ok, I am on it